enterprise-viewer/latest/install/index.md

```diff
@@ -255,9 +255,11 @@ You only need to follow these steps if installing AEV without ACA:
 
    > **Note:** Make sure you are using the correct `tsgrp-opencontent.amp` for your version of Alfresco.
 
-   * If using Alfresco Content Services 7.2.x, use the `tsgrp-opencontent-3.6-for-acs7.2.amp`.
-   * If using Alfresco Content Services 7.3.x, use the `tsgrp-opencontent-3.6-for-acs7.3.amp`.
-   * If using Alfresco Content Services 7.4.x, use the `tsgrp-opencontent-3.6-for-acs7.4.amp`.
+   For example:
+
+   * If using Alfresco Content Services 23.x, use the `tsgrp-opencontent-4.0-for-acs23.x.amp`.
+
+   where `x` is the specific Content Services version you're running.
```

Comment:
I've added a generic example here for 23.x. Need to verify if this is the correct file format in the release artefacts.